quickjs.h

```diff
@@ -335,6 +335,13 @@ JS_EXTERN void JS_AddIntrinsicBigInt(JSContext *ctx);
 JS_EXTERN void JS_AddIntrinsicWeakRef(JSContext *ctx);
 JS_EXTERN void JS_AddPerformance(JSContext *ctx);
 
+/* for equality comparisons and sameness */
+JS_EXTERN JS_BOOL JS_IsEqual(JSContext *ctx, JSValue op1, JSValue op2);
```

Comment:
Please change the return type for `JS_IsEqual` to `int`

Reply:
Thanks for your review suggestions, I've changed the return type for `JS_IsEqual` to `int`.